### DISCUSSION TITLE
Qodana - Azure DevOps and Service Connection

### DISCUSSION THREAD

**Discussion by nedimaf** [JetBrains/Qodana#245] [Q&A]
Hi everyone,

Is there any possibility to not use SSH and HTTPS credentials to connect Qodana and the project, but to use different approach? Is it possible to make a service connection between repository in Azure DevOps and Qodana cloud?

Thank you and best regards!

**Comment by brichbash** [comment]
Hi,

Currently, you can skip setting up VCS connection during trials, also, within this week we'll disable this requirement for other license types as well.
As for the question, at the moment there is no way to connect your repo to Cloud without an SSH/HTTPS connection.
